packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailOverview.tsx

```diff
@@ -66,13 +68,20 @@ export const AlertDetailOverview = React.memo((props: OverviewProps) => {
           label="Type"
           value={convertStringToCamelCasesWithSpaces(type)}
         />
+        <AlertDetailRow
+          label="Created"
+          value={formatDate(created, {
+            format: 'MMM dd, yyyy, h:mm a',
```

Comment:
Usually we pass the user's timezone so the date is corrected for the timezone the user has selected in their profile. Should we do this here?

```suggestion
            format: 'MMM dd, yyyy, h:mm a',
            timezone: profile?.timezone,
```

Reply:
thanks for pointing out, I've updated the same